packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/fixtures/go/lock-asset/chaincode-typescript/tslint.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@jscode017 This seems like it's not being used. If the latter is correct then please delete the file.

Reply:
done